automatic/chromium/update.ps1

```diff
@@ -1,74 +1,74 @@
+Import-Module au
 
-import-module au
-    $releases = 'https://chromium.woolyss.com/api/v5/?os=win<bit>&type=<type>&out=json'
-    $ChecksumType = 'sha256'
+$checksumType = 'sha256'
 
 function global:au_SearchReplace {
   @{
     ".\legal\verification.txt" = @{
-    "(?i)(\s*32\-Bit Software.*)\<.*\>"        = "`${1}<$($Latest.URL32)>"
-    "(?i)(\s*64\-Bit Software.*)\<.*\>"        = "`${1}<$($Latest.URL64)>"
-    "(?i)(^\s*checksum\s*type\:).*"            = "`${1} $($Latest.ChecksumType32)"
-    "(?i)(^\s*checksum32\:).*"                 = "`${1} $($Latest.Checksum32)"
-    "(?i)(^\s*checksum64\:).*"                 = "`${1} $($Latest.Checksum64)"
+      "(?i)(\s*32\-Bit Software.*)\<.*\>"        = "`${1}<$($Latest.URL32)>"
```

Comment:
Since we aren't embedding the 32bit binary anymore, this regex should be removed as well as the mention in the VERIFICATION.txt file (or at least updated to mention it is downloaded during runtime).